source/documentation/breaking-changes/random-with-units.md.erb

```diff
@@ -0,0 +1,36 @@
+---
+title: "Breaking Change: Random With Units"
+introduction: >
+  Sass `random($limit)` will no longer ignore `$limit` units in a future
```

Comment:
```suggestion
  The `random()` function will no longer ignore units on the `$limit` argument in a future
```

* We try to use "the NAME TYPE" when referring to Sass constructs, including functions and parameters of those functions. This has three benefits:

  * It allows us to avoid starting sentences with an uncapitalized code name.
  * If we're linking to those constructs' documentation, it means the link covers more than just the `<code>` tag which makes it more visible to readers.
  * It gives unfamiliar readers more context to hook onto.

* We usually refer to functions in documentation without parameters, since parameter lists get quite long for more complex functions like `color.scale()`.